omnibus/package-scripts/agent/posttrans

```diff
@@ -17,7 +17,7 @@ if [ -f "/etc/SuSE-release" ] || [ "$DISTRIBUTION" = "openSUSE" ] || [ "$DISTRIB
     DISTRIBUTION_FAMILY="SUSE"
 fi
 
-if [ "$DISTRIBUTION_FAMILY" == "SUSE" ]; then
+if [ "$DISTRIBUTION_FAMILY" = "SUSE" ]; then
```

Comment:
was this intentional ?

Reply:
Yes, apparently the previous syntax (which we use in lots of places) is not POSIX and the linter didn't like it.